Backend/controllers/auth.js

```diff
@@ -161,3 +169,31 @@ exports.logout = async (req, res, next) => {
     data: {},
   });
 };
+
+exports.createAdmin = async (req, res, next) => {
+  try {
+    const password = generator.generate({
+      length: 10,
+      numbers: true,
+    });
+    const { name, email, tel } = req.body;
+    // Create user
+    const user = await User.create({
+      name,
+      tel,
+      email,
+      password,
+    });
```

Comment:
user should have role `admin` 
```suggestion
      role: "admin",
    });
```